### ISSUE TITLE
PhantomJS Support

### ISSUE THREAD

**Issue by wogsland** [Polymer/project#20]
I know this was brought up a couple years ago [elsewhere](https://github.com/Polymer/polymer/issues/724), but I am curious where this stands on the current roadmap.

**Comment by tjsavage** [comment]
Do you happen to have a specific case that's failing with PhantomJS? To the extent that the version of Webkit that's currently bundled with PhantomJS is a Polymer supported browser, it should *just work*. This is likely hard to say for sure, as I know PhantomJS webkit feature support doesn't necessarily correlate directly to a Safari version.

Testing out a couple Polymer 1.x sites with PhantomJS just now, I'm seeing correct rendering results. It's entirely possible that the jump from 0.5 to 1.0 fixed a bunch of PhantomJS issues - the blog post linked from that older issue would have been using pre-1.0.

**Comment by wogsland** [comment]
Yes, I posted more details about my specific case [on stackoverflow](http://stackoverflow.com/questions/35610392/cant-get-website-image).

**Comment by tjsavage** [comment]
Got it, thanks! From poking around a bit I don't see anything obviously breaking, just that the capture is off - there's a lot on that page though that could be interacting funnily.  A minimal reproduction of PhantomJS not rendering a Polymer page correctly would be ideal to help debug.

It might be interesting to see if the https://github.com/webcomponents/webcomponentsjs/ tests pass when run on the PhantomJS headless browser.
